contrib/bundle/test-e2e

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Check for root
+if [[ $EUID -ne 0 ]]; then
+    echo "Please run this script as root"
+    exit 1
+fi
+
+# Assume we're running on this arch
+ARCH=amd64
+
+# Global vars to be used
+# shellcheck source=vars
+source "$(dirname "${BASH_SOURCE[0]}")"/vars
+
+pushd "$ARCHIVE_PATH"
+
+# Untar the bundle
+tar xfvz "$ARCHIVE"
+pushd cri-o
+
+# Install and prepare config
+./install
+
+cat <<EOT >>/etc/crio/crio.conf.d/30-cgroup-manager.conf
+[crio.runtime]
+conmon_cgroup = "pod"
+cgroup_manager = "cgroupfs"
+EOT
+
+# Start CRI-O
+systemctl daemon-reload
+systemctl start crio
+systemctl is-active --quiet crio && echo CRI-O is running
+
+# Start Kubernetes
+GOROOT="$GOROOT_1_16_X64"
+GOPATH="$HOME/go"
+PATH="$GOROOT/bin:$PATH"
+K8SPATH="$GOPATH/src/k8s.io"
+mkdir -p "$K8SPATH"
+pushd "$K8SPATH"
+git clone --depth=1 https://github.com/kubernetes/kubernetes
+pushd kubernetes
+
+echo Downloading server binaries
+export KUBERNETES_SKIP_CONFIRM=1
+export KUBERNETES_SKIP_CREATE_CLUSTER=1
+cluster/get-kube.sh
+
+mkdir -p _output/bin
+tar xfz kubernetes/server/kubernetes-server-linux-amd64.tar.gz
+cp kubernetes/server/bin/{kubectl,kube-apiserver,kube-controller-manager,kube-proxy,kube-scheduler,kubelet} \
+    _output/bin
+
+echo Downloading test binaries
+LATEST=$(curl -sSfL https://dl.k8s.io/ci/latest.txt)
+curl -sSfL "https://dl.k8s.io/ci/$LATEST/kubernetes-test-linux-amd64.tar.gz" \
+    -o- | tar xfz -
+cp kubernetes/test/bin/{ginkgo,e2e.test} _output/bin
+
+export CONTAINER_RUNTIME=remote
+export CGROUP_DRIVER=cgroupfs
```

Comment:
should be `systemd`?

Reply:
Complete set of node args we used in the prow job for node e2e, https://github.com/kubernetes/test-infra/blob/927be10bc7aee8a8eed3d97cd4286ccd09dea4cb/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L334

Reply:
Tried to set the args in an additional commit.